lighthouse-core/audits/is-on-https.js

```diff
@@ -74,18 +82,42 @@ class HTTPS extends Audit {
           .filter(record => !HTTPS.isSecureRecord(record))
           .map(record => URL.elideDataURI(record.url));
 
+      /** @type {Array<{url: string, resolution?: string}>}  */
       const items = Array.from(new Set(insecureURLs)).map(url => ({url}));
 
-      let displayValue = '';
-      if (items.length > 0) {
-        displayValue = str_(UIStrings.displayValue, {itemCount: items.length});
-      }
-
       /** @type {LH.Audit.Details.Table['headings']} */
       const headings = [
         {key: 'url', itemType: 'url', text: str_(UIStrings.columnInsecureURL)},
       ];
 
+      // Mixed-content issues aren't emitted until M84.
+      if (artifacts.InspectorIssues.mixedContent.length) {
+        headings.push(
+          {key: 'resolution', itemType: 'text', text: str_(UIStrings.columnResolution)});
+        for (const details of artifacts.InspectorIssues.mixedContent) {
+          let item = items.find(item => item.url === details.insecureURL);
+          if (!item) {
+            item = {url: details.insecureURL};
```

Comment:
just curious: are any of the `SECURE_SCHEMES` from above ever going to show up here? Maybe under warning?